### DISCUSSION TITLE
/login : 404 Not Found

### DISCUSSION THREAD

**Discussion by chestserva** [ikhsan3adi/absensi-sekolah-qr-code#20] [Q&A]
public $views = [
        'login'           => '\App\Views\admin\login',
        'register'        => 'Myth\Auth\Views\register',
        'forgot'          => 'Myth\Auth\Views\forgot',
        'reset'           => 'Myth\Auth\Views\reset',
        'emailForgot'     => 'Myth\Auth\Views\emails\forgot',
        'emailActivation' => 'Myth\Auth\Views\emails\activation',
    ];

login page 404 not found, mohon pencerahannya suhu

**Comment by pratamarvi** [comment]
sama, ini gimana ngatasinnya, padahal udah sama tapi tidak ditemukan

**Comment by pratamarvi** [comment]
Not Found
The requested URL was not found on this server.

**Comment by ikhsan3adi** [comment]
Coba lihat kodingan fullnya di file itu
Sama screenshotnya
